### PR TITLE
Azure PS Tasks - update openssl package with config file

### PR DESCRIPTION
**Pull request by vincentdass**
Fixes: #10335

### CODE REVIEW

File: Tasks/AzureCloudPowerShellDeploymentV1/task.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 3,
-        "Patch": 16
+        "Minor": 153,
```
Comment (kmkumaran) [inline Tasks/AzureCloudPowerShellDeploymentV1/task.json]: This is a huge jump in Minor version...

File: Tasks/AzureFileCopyV1/task.json [hunk 2]
```diff
@@ -13,8 +13,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 0,
-        "Patch": 151
+        "Minor": 153,
```
Comment (kmkumaran) [inline Tasks/AzureFileCopyV1/task.json]: Same here.

File: Tasks/AzureFileCopyV2/task.json [hunk 3]
```diff
@@ -13,8 +13,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 1,
-        "Patch": 9
+        "Minor": 153,
```
Comment (kmkumaran) [inline Tasks/AzureFileCopyV2/task.json]: It seem to be present in all tasks.

File: Tasks/Common/VstsAzureHelpers_/Utility.ps1 [hunk 4]
```diff
@@ -225,6 +225,9 @@ function ConvertTo-Pfx {
     }
 
     $openSSLExePath = "$PSScriptRoot\openssl\openssl.exe"
+    $env:OPENSSL_CONF = "$PSScriptRoot\openssl\openssl.cnf"
```
Comment (kmkumaran) [inline Tasks/Common/VstsAzureHelpers_/Utility.ps1]: Is this fix required only for PS based task?

Comment (vincentdass) [reply Tasks/Common/VstsAzureHelpers_/Utility.ps1]: yes. there is no issue in node based tasks

File: Tasks/AzureCloudPowerShellDeploymentV1/task.json (hunk 1 above)
Comment (kmkumaran) [reply Tasks/AzureCloudPowerShellDeploymentV1/task.json]: Ok. I see you are aligning this with sprint.

**Pull request merged**